spec/migrations/add_thermometers_to_old_fundraisers_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+migration_file_name = Dir[Rails.root.join('db/migrate/*_add_thermometers_to_old_fundraisers.rb')].first
+require migration_file_name
```

Comment:
You an require the `Rails.root.join('db/migrate/*_add_thermometers_to_old_fundraisers.rb')` directly so no need for `Dir[...].first`

Reply:
Changed :)